README.adoc

```diff
@@ -0,0 +1,100 @@
+*** 3. create a PR
+** *Your PR requires changes in https://github.com/codeready-toolchain/host-operator[host-operator] repo but not in https://github.com/codeready-toolchain/member-operator[member-operator] repo:*
+*** 1. check the name of a branch you are going to create a PR for
+*** 2. create a branch with the same name within your fork of https://github.com/codeready-toolchain/host-operator[host-operator] repo and put all necessary changes there
+*** 3. make sure that your fork of https://github.com/codeready-toolchain/member-operator[member-operator] repo doesn't contain a branch with the same name
+*** 4. push all changes into both forks of the repositories https://github.com/codeready-toolchain/toolchain-e2e[toolchain-e2e] and https://github.com/codeready-toolchain/host-operator[host-operator]
+*** 5. create a PR for https://github.com/codeready-toolchain/toolchain-e2e[toolchain-e2e]
+*** 6. create a PR for https://github.com/codeready-toolchain/host-operator[host-operator]
+** *Your PR requires changes in https://github.com/codeready-toolchain/member-operator[member-operator] repo but not in https://github.com/codeready-toolchain/host-operator[host-operator] repo:*
+*** See the previous case and just swap member-operator and host-operator.
+** *Your PR requires changes in both repos https://github.com/codeready-toolchain/host-operator[host-operator] and https://github.com/codeready-toolchain/member-operator[member-operator]:*
+*** 1. check the name of a branch you are going to create a PR for
+*** 2. create a branch with the same name within your fork of https://github.com/codeready-toolchain/host-operator[host-operator] repo and put all necessary changes there
+*** 3. create a branch with the same name within your fork of https://github.com/codeready-toolchain/member-operator[member-operator] repo and put all necessary changes there
+*** 4. push all changes into all your forks
+*** 5. create a PRs for all repos https://github.com/codeready-toolchain/toolchain-e2e[toolchain-e2e], https://github.com/codeready-toolchain/host-operator[host-operator] and https://github.com/codeready-toolchain/member-operator[member-operator]
+
+=== Verifying the OpenShift CI configuration
+
+ It's possible to verify the OpenShift CI config from the developer's laptop while all the jobs are executed on the remote, online CI platform:
+
+1. checkout and build the https://github.com/openshift/ci-tools[CI Operator] command line tool
+2. login to https://console.svc.ci.openshift.org (via GH OAuth) and copy the login command (you may need to switch to the `application console`)
+3. login with the command aforementioned
+4. run the CI jobs with
++
+```
+ci-operator --config ../../openshift/release/ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml --git-ref=codeready-toolchain/toolchain-e2e@master
+```
+
+assuming that the https://github.com/openshift/release[OpenShift Release] repo was checked you.
```

Comment:
> assuming that the https://github.com/openshift/release[OpenShift Release] repo was checked you. 

seems like part of this sentence is missing?

Reply:
copy-pasted from other repo ;-) 
fixed 463d37f